src/molecules/account-menu.module.css

```
@@ -0,0 +1,29 @@
+/* AccountMenu.module.css */
+.accountMenu {
```

```suggestion
.accountMenu {
    position: absolute;
    min-width: 112px;
    top: 16px;
    right: 16px;
    display: flex;
    flex-direction: column;
```
このようにして画面上に浮かべて, 最小幅を指定して, 中の要素を縦に並べるとメニューっぽく機能するかと思います.